packages/core/src/client.ts

```diff
@@ -289,6 +289,25 @@ export const Client: new (opts: ClientOptions) => Client = function Client(
     },
 
     createRequestOperation(kind, request, opts) {
+      if (
+        kind !== 'teardown' &&
+        !request.query.definitions.some(
+          x => x.kind === Kind.OPERATION_DEFINITION && x.operation === kind
+        ) &&
+        process.env.NODE_ENV !== 'production'
+      ) {
+        for (let i = 0; i < request.query.definitions.length; i++) {
```

Comment:
`.find` wasn't allowed with the `ES5` lint-rule 😅

Reply:
The `some` combined with the extra loop seems a bit... insufficient even if it's just for development. So we could probably add an operation type helper here that's similar to the name one: https://github.com/FormidableLabs/urql/blob/30447ec77a8c44e0585db5af4febf58e1c253f35/packages/core/src/utils/request.ts#L93